examples/island/Island.cpp

```diff
@@ -161,8 +162,8 @@ int main()
     statusText.setPosition({(windowWidth - statusText.getLocalBounds().width) / 2.f, (windowHeight - statusText.getLocalBounds().height) / 2.f});
 
     // Set up an array of pointers to our settings for arrow navigation
-    Setting settings[] =
-    {
+    std::array<Setting, 9> settings =
```

Comment:
This array can also be constexpr.

Reply:
Oh yeah you're right! I didn't mark that constexpr because I saw the following lines farther down:
```cpp
case sf::Keyboard::Left:  *(settings[currentSetting].value) -= 0.1f; break;
case sf::Keyboard::Right: *(settings[currentSetting].value) += 0.1f; break;
```
It almost appears as if the array is being mutated but only some non-const pointed-to values are being changed. Thank you!